ompi/mca/osc/ucx/osc_ucx.h

```diff
@@ -150,6 +162,17 @@ typedef struct ompi_osc_ucx_lock {
 #define OSC_UCX_GET_EP(comm_, rank_) (ompi_comm_peer_lookup(comm_, rank_)->proc_endpoints[OMPI_PROC_ENDPOINT_TAG_UCX])
 #define OSC_UCX_GET_DISP(module_, rank_) ((module_->disp_unit < 0) ? module_->disp_units[rank_] : module_->disp_unit)
 
+extern bool mpi_thread_multiple_enabled;
+
+#define OSC_UCX_GET_DEFAULT_EP(_ep_ptr, _comm, _target)                 \
+    if (mpi_thread_multiple_enabled) {                                  \
+        _ep_ptr = NULL;                                                 \
+    } else {                                                            \
+        _ep_ptr = (ucp_ep_h *)&(OSC_UCX_GET_EP(_comm, _target));        \
+    }
+
+extern int outstanding_ops_flush_threshold;
```

Comment:
To make @drwootton life easier, this should be properly prefixed with `ompi_osc_ucx_` :)

Reply:
done

Reply:
Thanks, Jeff for the tip. Will do.